modules/KalturaSupport/resources/webcast/userProfile.js

```diff
@@ -0,0 +1,75 @@
+(function (mw, $) {
```

Comment:
This component was created based on the one that is currently in use in qNa so it suppose to be fine.